src/Installers/Windows/RuntimeSiteExtension/Microsoft.AspNetCore.Runtime.SiteExtension.csproj

```
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(RepositoryRoot)\.deps\dependencies.g.props" />
+  <Import Project="..\..\Directory.Build.props" />
```

```suggestion
  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory)..\, Directory.Build.props))\Directory.Build.props" />
```

Automatically follow the chain up to the nearest DB.props file.